mapillary_tools/geotag/geotag_images_from_nmea_file.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         )
 
 
-def get_lat_lon_time_from_nmea(nmea_file: Path) -> T.List[geo.Point]:
+def get_lat_lon_time_from_nmea(nmea_file: Path) -> T.Sequence[geo.Point]:
```

Comment:
Not sure if it's expected: returning `Sequence` makes it immutable

Reply:
Thanks! Reverting